libs/langchain-community/src/embeddings/ollama.ts

```diff
@@ -21,7 +21,7 @@ interface OllamaEmbeddingsParams extends EmbeddingsParams {
   headers?: Record<string, string>;
 
   /** Defaults to "5m" */
-  keepAlive?: string;
+  keepAlive?: string | number;
```

Comment:
The community implementations have been deprecated in favor of the `@langchain/ollama` package. 

```suggestion
  keepAlive?: string;
```